.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,66 +1,32 @@
-<!--
```

Comment:
These comments are meant to guide folks unfamiliar with our code, content, and review standards. They're not visible in the description itself if not deleted, so I'm wary removing them. Would you mind moving the guidelines into a new doc in our `documentation` folder and linking to in the PR template?

Reply:
I shortened all the comments where they were actually useful, and only removed them where they were redundant. For example the PR title just gives an example title, but we can remove the comments in `WHAT` and `WHY` because those are self-evident

Reply:
Gotcha (just viewed the file directly so I could better see what was left) 👍 The template def has gotten to be a little all over the place, all these comments were added ad hoc in response to repeat questions/feedback but without editing for big picture. If the contribution projects end up getting picked back up, a thorough contribution guide will ship as part of that effort.